src/components/PostInfo/PostInfo.tsx

```diff
@@ -1,5 +1,46 @@
 import React from 'react';
+import './PostInfo.scss';
+import { FullPost } from '../../types/Post';
+import { CommentList } from '../CommentList';
+import { UserInfo } from '../UserInfo';
 
-export const PostInfo: React.FC = () => (
-  <>Put the post here</>
-);
+type Props = {
+  post: FullPost,
+};
+
+export const PostInfo: React.FC<Props> = ({ post }) => {
+  const {
+    title,
+    body,
+    user,
+    comments,
+  } = post;
+
+  return (
+    <div className="PostInfo">
+      <div className="PostInfo__header">
+        <h3 className="PostInfo__title">{title}</h3>
+
+        { user && (
+          <p>
+            {' Posted by  '}
+            <UserInfo user={user} />
+          </p>
```

Comment:
```suggestion
          <UserInfo user={user} />
```
move it to UserInfo component